src/Server.cpp

```diff
@@ -1148,9 +1132,10 @@ void handleBody(Server& server, int activeFd, Connection& connection)
 	if (isCompleteBody(connection)) {
 		LOG_DEBUG << "Received complete request body";
 		// Printing body can be confusing for big files.
-		//LOG_DEBUG << connection.m_buffer;
+		// LOG_DEBUG << connection.m_buffer;
 		try {
-			server.parseBody(connection.m_buffer, connection.m_request);
+			std::vector<char>& buffer = server.getBuffer();
+			server.parseBody(connection.m_buffer, connection.m_request, buffer);
```

Comment:
If we don't pass directly in dispatch we could pass server.getBuffer() without intermediate variable:
`server.parseBody(connection.m_buffer, connection.m_request, server.getBuffer() );`